spec/features/user_can_see_book_review_spec.rb

```diff
@@ -0,0 +1,90 @@
+require 'rails_helper'
+
+describe 'book reviews' do
+  context 'When a user visit a book show page' do
+    it  'see the book title and a list of reviews for that book' do
+      book = Book.create(title: 'Left hand of darkness')
+      user = User.create(name: "Joe")
+      review = book.reviews.create(body: "it was good", rating: 4, user_id: user.id, book_id: book.id)
+
+      visit "/books/#{book.id}"
+
+      expect(page).to have_content("Left hand of darkness")
+      expect(page).to have_content("it was good")
+      expect(page).to have_content(4)
+
+
+    end
+
+    #
+    it  'sees the user that created the review' do
+      book = Book.create(title: 'Left hand of darkness')
+      user = User.create(name: "Joe")
+      review1 = book.reviews.create(body: "it was bad", rating: 2, user_id: user.id, book_id: book.id)
+      review2 = book.reviews.create(body: "it was alright", rating: 4, user_id: user.id, book_id: book.id)
+      review3 = book.reviews.create(body: "it was good", rating: 6, user_id: user.id, book_id: book.id)
+
+      visit "/books/#{book.id}"
+
+
+      expect(page).to have_content("#{user.id}")
```

Comment:
I would probably rather see the user's name here. You could call `review.user.name` to get that in your view.